client/src/components/Header/components/universal-nav.css

```diff
@@ -348,7 +348,7 @@ button.nav-link[aria-disabled='true'] {
 }
 
 .toggle-button-nav:focus:not(:focus-visible) {
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert back to `outline: none`. This is for non-keyboard focus in which we don't want a focus outline.